### PR TITLE
Awesome books: refactor to use JavaScript classes

### PR DESCRIPTION
**Pull request by david63011**
Learning objectives
Use JavaScript classes.
Understand how to use medium-fidelity wireframes to create a UI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,30 @@
...
+
+GitHub: @david63011
+Twitter: @david630
+Show your support
+Give a ⭐️ if you like this project!
+
+# License
+
+This project is MIT licensed.
+
```
Comment (BertrandConxy) [inline README.md]: - [ ] Good job so far 👏. Your project works fine. However, you need to have good documentation too. In order to make your README professional, Microverse provided a [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) that you are advised to use for all your project. Please customize it to your project and make sure that all the sections are covered 😉

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @david63011 , 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 

## Highlights 💪 
- No linter errors
- correct flow is used
- good PR


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (david63011) [reply README.md]: ok I updated it

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @david63011,

## Status: APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 
- Great job with your design and functionalities 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**